examples/12. Inline Action.md

```diff
@@ -0,0 +1,66 @@
+FORMAT: 1A
+
+# Inline Action API
+This API example demonstrates that a resource can contain an in-place definition of another resource. This allows us to show that the resource `Task` can afford an action that is not limited to the resource `Task`.
```

Comment:
> resource can contain an in-place definition of another resource

This is actually not correct statement. Again, this should be talking about arbitrary state transitions (precisely – state transitions that are not limited to the parent resource as its target)